keyboards/gmmk/gmmk2/p96/iso/keymaps/vial/vial.json

```
@@ -0,0 +1,252 @@
+{
+  "name": "GMMK V2 96 ISO",
+  "vendorId": "0x320F",
+  "productId": "0x505A",
+  "matrix": {
+    "rows": 14,
+    "cols": 8
+  },
+  "keycodes": [
+    "qmk_lighting"
+  ],
+  "menus": [
+    "qmk_rgb_matrix"
+  ],
```

Since you have `VIALRGB_ENABLE = yes` in the `vial` keymap's `rules.mk`, this should really be changed to 
```suggestion
"lighting": "vialrgb",
```